### ISSUE TITLE
Octave 4 Error

### ISSUE THREAD

**Issue by matejvelikonja** [Gutza/octave-daemon#20]
Has anyone tested this library with Octave 4?

I'm receiving a strange error at https://github.com/Gutza/octave-daemon/blob/master/include/classes/Octave_controller.php#L315

with message 

```
[ErrorException]
  Undefined offset: 0
```

**Comment by matejvelikonja** [comment]
and if I comment out the problematic block I receive

```
[ErrorException]
Octave: octave: X11 DISPLAY environment variable not set
  octave: disabling GUI features
```

**Comment by matejvelikonja** [comment]
Ok, I think I've fixed in #21. Also works on older octave.

Please merge.
